lib/worker/base.js

```diff
@@ -176,7 +176,21 @@ const run = async options => {
 
 	try {
 		for await (const ref of (options.require || [])) {
-			await load(ref);
+			if (typeof ref === 'string') {
+				await load(ref);
+			} else if (Array.isArray(ref)) {
+				const [path, options] = ref;
+
+				const loadedModule = await load(path);
+
+				if (typeof loadedModule === 'function') {
+					await loadedModule.apply(...options);
```

Comment:
This uses the first option as the `this` argument, so you'd always have to pass `null` which seems odd. I think it's better like:

```suggestion
					await loadedModule(...options);
```